lib/capybara/webkit/matchers.rb

```diff
@@ -7,11 +7,20 @@ module RspecMatchers
           actual.error_messages.any?
         end
 
-        failure_message_for_should do |actual|
+        # Check that RSpec is < 2.99
+        if !respond_to?(:failure_message) && respond_to?(:failure_message_for_should)
+          alias :failure_message :failure_message_for_should
+        end
+
+        if !respond_to?(:failure_message_when_negated) && respond_to?(:failure_message_for_should_not)
+          alias :failure_message_when_negated :failure_message_for_should_not
+        end
+
+        failure_message do |actual|
```

Comment:
Unused block argument - `actual`. You can omit the argument if you don't care about it.